include/gatekeeper_gk.h

```diff
@@ -21,16 +21,131 @@
 
 #include <rte_atomic.h>
 
+#include "gatekeeper_net.h"
 #include "gatekeeper_ipip.h"
 #include "gatekeeper_ggu.h"
 #include "gatekeeper_mailbox.h"
+#include "gatekeeper_lpm.h"
+
+/*
+ * The LPM supports 8-bit next hop,
+ * which at most has 256 different values.
+ */
+#define GK_MAX_NUM_FIB_ENTRIES (256)
```

Comment:
The comment is outdated. `struct rte_lpm_tbl_entry` in `dependencies/dpdk/lib/librte_lpm/rte_lpm.h` reservers 24 bits for next hop. Please update the comment with the following one:
```
/*
 * The LPM reserves 24-bit for the next-hop field.
 * TODO Drop the constant below and make it dynamic.
 */
```